aten/src/ATen/native/cuda/SummaryOps.cu

```diff
@@ -0,0 +1,282 @@
+#include "ATen/ATen.h"
+#include "ATen/cuda/CUDAApplyUtils.cuh"
+
+namespace at {
+
+namespace cuda { namespace detail {
```

Comment:
why is this not under `at::native`?

Reply:
I'm following the convention of `CUDAApplyUtils.cuh` - removing `namespace detail` and just under `at::cuda`.